Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:alpine
```

Comment:
i am also working on docker support. my dockerfile looks a little different. maybe lets merge to best of those to toger?
```Dockerfile
FROM node:9.3-alpine
WORKDIR /home/node

# adding package.json first, to take advantage of cached docker layers
ADD package.json package.json
RUN npm install --only=production && npm cache clean --force
USER node

# add the rest of the app
COPY . .
EXPOSE 8099

CMD ["npm", "start"]
```

Reply:
Sure, but what would you like to change?
I think it's best to use latest node to avoid being stuck with a version with vulnerabilities once they are disclosed.
`npm cache clean` is discouraged since npm 5 and not useful here.
As for the `WORKDIR` and `USER`, it doesn't really matter so I went simple here.

Reply:
> FROM node-VERSION

you are right. for work i would prefer a specific version to get reproducable builds, but in a project like this always having the latest version is perfectly fine.

> USER node

Per default the app would run as the container-root, which might or might not be an issue. but since they added a node user to their base image why not just use it. together with its home dir. but i agree, not much of a difference here

Reply:
> npm cache clean --force

this results in slightly smaller images. so i see no harm doing it in a docker-file. since it will be the last npm command ever made ;O